src/app/index.html

```diff
@@ -14,167 +14,77 @@
-        "title": "Layout",
-        "inputs": [
-        {
-          "id": "circular_layout",
-          "text": "Has the circular layout been applied?",
-          "type": "checkbox"
-        }]
-        }-->
-        </div>
-      </section>
-      <section id="session_save">
-        <h2>Session Save</h2>
-        <p class="text">Saving Cytoscape session file...</p>
-      </section>
-      <section id="runjasmine">
-        <div class="cyrest">
-          <!--{
-          "title": "Ready to run automated tests!",
-          "text": "The test results will be shown in the bottom panel"
-          }-->
-        </div>
-      </section>
-      <section id="user_feedback">
-        <div class="cyrest">
-          <!--{
-          "title": "Go through personal workflows!",
-          "text": "Ensure everything works as you expect.",
-          "inputs": [{
-            "id" : "feedback",
-            "type" : "textarea",
-            "text" : "Feedback"}
-            ]
-          }-->
-        </div>
-      </section>
-      <section id="close_cytoscape">
-        <h2>Thanks for testing Cytoscape!</h2>
-        <p class="text">Please submit yout framework.log</p>
-      </section>
-      <section id="submit">
-        <div class="cyrest">
-          <!--{
-            "title": "Submit your testing results",
-            "text": ""
-          }-->
-        </div>
-      </section>
-    </div>
-  </div>
-</div>
-  <div id="tests">
-  </div>
-  <script src="lib/js/head.min.js"></script>
-  <script src="js/reveal.js"></script>
-  <script src="js/main.js"></script>
-</body>
+		<script src="js/cycaller.js"></script>
+		<script src="js/slides/session.js"></script>
+		<script src="js/slides/slidesModel.js"></script>
+		<script src="spec/cytoscape_spec.js"></script>
+		
+		<!-- Printing and PDF exports -->
+		<script>
+			var link = document.createElement( 'link' );
+			link.rel = 'stylesheet';
+			link.type = 'text/css';
+			link.href = window.location.search.match( /print-pdf/gi ) ? 'css/print/pdf.css' : 'css/print/paper.css';
+			document.getElementsByTagName( 'head' )[0].appendChild( link );
+		</script>
+	</head>
+	<body>
+		
+		<div class="reveal">
+			<div class="slides">
+				<section>
+					<h2>CYTOSCAPE TESTING</h2>
+					<div class="row">
+						<div class="column">
+						<p class="paragraphText">Thank you for helping improve the Cytoscape experience.  Use the arrows at the bottom to step through the test.</p>
+						<div id="log-div">
+							<div id="log-button-container">
+							<button type="button" id="log-button" onClick="toggleLog()" class="btn btn-primary" value="LOG">Log</button>
+							<!-- <input type="button" id="log-button" onClick="toggleLog()" value="LOG"/> -->
+							</div>
+							<div id="log-container" style="display:none">
+							<textarea id="log"></textarea>
+							</div>
+						</div>
+						</div>
+							<div class="column right">
+								<div>
+									<label class="labelText">Tester Name:</label>
```

Comment:
Let's move tester name field to the bottom of welcome text, it doesn't look good to me. Also log button needs to have a better position on the screen, I believe initial position was better.